server/config/game/specialists.json

```diff
@@ -797,19 +799,19 @@
             "id": 16,
             "key": "rocket-thruster",
             "name": "Stellar Engine",
-            "description": "When the Stellar Engine is activated, the star slowly moves towards the nearest Stellar Beacon at x0.25 carrier speed.",
+            "description": "When the Stellar Engine is activated, the star slowly moves towards the nearest Stellar Beacon at 10% of carrier speed.",
```

Comment:
I feel in the last play test that it was pretty balanced as it was. It really doesn't add much, and 0.25 is plenty slow as it is. Making it slower _and_ twice the price would make it pretty worthless.

Reply:
What about upping the cost slightly to 7 and decreasing the speed somewhat to 20%? 

The idea of moving an entire star system should feel slow, and most games do last long enough where it's still impactful. The physics involved in moving a star and all its planets vs. a carrier should be reinforced through the mechanics. The fact the star moves to the stellar beacon in a straight line and doesn't need to follow hyperspace lane connection points between nearby stars, gives it a built-in speed increase. I find that the carriers factoring in having to follow hyperspace lanes and needing hyperspace range tech slows them down as they need to find connection routes, and makes it hard to catch a star that has been hijacked.